frontend/prebuild/src/app/login/login.component.html

```diff
@@ -32,6 +32,9 @@ <h2 class="section-header">or</h2>
             <div class="form-item">
               <button type="button" (click)="loginAsGuest(username)">Sign In</button>
             </div>
+            <div class="form-item">
+              <button type="button" (click)="loginGoogle()">Login Through Google</button>
```

Comment:
Can you make this say "Sign In With Google" for consistency? And we can put this above the guest button. That button should probably be renamed "Sign In As Guest" now that there's an alternative.

Reply:
yes